cppcache/static/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ project(apache-geode-static LANGUAGES CXX)
 
 add_library(apache-geode-static STATIC ${SOURCES_ALL})
 
+generateExportFile( "apache-geode-static" )
```

Comment:
Technically this doesn’t need quoting. Also it would be nice to follow the naming convention of CMake.

Reply:
Fixed to follow the cmake function naming convention. I realize cmake doesn't required the quotes in this case, but I prefer them.